qemu/tests/sr_iov_hotplug.py

```diff
@@ -166,36 +207,40 @@ def _find_pci():
 
             if not utils_misc.wait_for(_find_pci, test_timeout, 3, 3):
                 test.fail("New add device not found in guest. "
-                          "Command was: %s" % find_pci_cmd)
+                          "Command was: lspci -nn")
 
             # Assign static IP to the hotplugged interface
             if params.get("assign_static_ip", "no") == "yes":
-                cmd = []
+                cmd = "service networking restart"
```

Comment:
%s/networking/network   ?

Reply:
Other parts look good to me. 
I tried to run the case on my host several times, but kept hitting the "login timeout" error. 
As long as @balamuruhans' tests pass, I think it is ok to merge.  
BTW, I'd prefer to run the test not only with Ubuntu if possible. Thanks!

Reply:
@huangyum Thanks! But if you observe in Line 232 I reassign the `cmd = service network restart` as there is a distro check in Line 231.

Reply:
@huangyum Also I have tested it and couldn't hit "login timeout" issue.